timemachine/cpp/src/potential.hpp

```diff
@@ -10,6 +10,8 @@ class Potential {
 public:
     virtual ~Potential(){};
 
+    static const int D = 3;
```

Comment:
may want to either give this a suffix D_ (to indicate its a class variable, even though its static), or namespace above with Potential::D (unscoped single letter variables can be pretty hard to track donw).

Reply:
Hmm, what about something like `ndim` to move away from single-level names with class scope?

> may want to either give this a suffix D_

Would that still be conventional if `D` is public? (in cases I've seen so far the trailing `_` is used for private variables -- we could also just make `D` private).

Reply:
`D` is fine too. Then maybejust modify the usages to `Potential::D` so scoping is clear? C++/C has the unfortunate behavior where any un-namespaced include (and their includes) is globally visible, so as a sanity let's explicit use the namespace?

Reply:
Makes sense. Would something like this work to make the scoping clear while minimizing line noise? df9bfda